test/splitEvery.js

```diff
@@ -0,0 +1,34 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('splitEvery', function() {
+
+  it('splits a collection into slices of the specified length', function() {
+    assert.deepEqual(R.splitEvery(1, [1, 2, 3, 4]), [[1], [2], [3], [4]]);
+    assert.deepEqual(R.splitEvery(2, [1, 2, 3, 4]), [[1, 2], [3, 4]]);
+    assert.deepEqual(R.splitEvery(3, [1, 2, 3, 4]), [[1, 2, 3], [4]]);
+    assert.deepEqual(R.splitEvery(4, [1, 2, 3, 4]), [[1, 2, 3, 4]]);
+    assert.deepEqual(R.splitEvery(5, [1, 2, 3, 4]), [[1, 2, 3, 4]]);
+    assert.deepEqual(R.splitEvery(3, []), []);
+  });
+
+  it('dispatches to `slice` method', function() {
```

Comment:
how they are derived i don't mind. exposing how they are derived and cementing that implementation in place with tests that i find questionable at least.